scripts/bumpSdkVersion.ts

```diff
@@ -0,0 +1,74 @@
+import { spawn } from "child_process";
+import fs from "fs/promises";
+import { glob } from "glob";
+import path from "path";
+
+const versionRegex = /^version\s*=\s*"[^"]*"/m;
+
+async function main() {
+  const newVersion = process.argv[2];
+  if (newVersion?.startsWith("v") !== true) {
+    console.log("Usage: bumpSdkVersion.ts <version>");
+    console.log("Version must start with 'v'");
+    process.exit(1);
+  }
+
+  // Remove the 'v' prefix for the actual version string
+  const versionNumber = newVersion.slice(1);
+
+  // Find all Cargo.toml files from workspace root
+  const workspaceRoot = path.resolve(__dirname, "..");
+  const cargoFiles = await glob("**/Cargo.toml", {
+    ignore: ["**/target/**", "**/node_modules/**"],
+    cwd: workspaceRoot,
+    absolute: true,
+  });
+
+  let success = true;
+  for (const cargoFile of cargoFiles) {
+    console.log(`Checking ${cargoFile}...`);
+    const content = await fs.readFile(cargoFile, "utf8");
+
+    if (!versionRegex.test(content)) {
+      console.log(`  ℹ️ Skipped, does not contain version field`);
+      continue;
+    }
+
+    // Only update the main version field, not dependencies
+    const updatedContent = content.replace(versionRegex, `version = "${versionNumber}"`);
```

Comment:
Should we enforce that the version number increments here?

Reply:
good idea